scripts/conp_to_nidm_terms/jsonld_generator.py

```diff
@@ -0,0 +1,43 @@
+import getopt
+from sys import argv
+from functions import generate_jsonld_files, collect_values
+
+
+def main(argv):
+    opts, args = getopt.getopt(argv, "", ["privacy=", "types=", "licenses=",
+                                          "is_about=", "formats=", "keywords=",
+                                          "use_api=", "help"])
+
+    options = dict(privacy=True, types=True, licenses=True, is_about=True, formats=True, keywords=True)
+    use_api = True
+
+    for opt, arg in opts:
+        opt_properties = ["--privacy", "--types", "--licenses", "--is_about", "--formats", "--keywords"]
+        if opt in opt_properties and arg == "False":
+            options[opt.replace("--", "")] = False
+        elif opt == '--use_api' and arg == "False":
+            use_api = False
+        else:
+            help_info()
+            exit()
```

Comment:
could maybe add a check here to see if the api_key has been set so that we do not have to wait after the report is created to know that we did not set the key.

Currently, if I run the script without having set the api_key, I get the following:
```
DATS files processed: 51
Traceback (most recent call last):
  File "jsonld_generator.py", line 43, in <module>
    main(argv[1:])
  File "jsonld_generator.py", line 33, in main
    generate_jsonld_files(report=report, use_api=use_api)
  File "/Users/cmadjar/GitHub/CONP-PCNO/conp-dataset/scripts/conp_to_nidm_terms/functions.py", line 172, in generate_jsonld_files
    jsonld_description["sameAs"] = get_api_response(term.lower())
  File "/Users/cmadjar/GitHub/CONP-PCNO/conp-dataset/scripts/conp_to_nidm_terms/functions.py", line 38, in get_api_response
    key = api_key_json["api_key"] if api_key_json["api_key"] != "" else _raise_error(f"{api_key_json['_comment']}")
  File "/Users/cmadjar/GitHub/CONP-PCNO/conp-dataset/scripts/conp_to_nidm_terms/functions.py", line 25, in _raise_error
    def _raise_error(er): raise Exception(er)
Exception: Add your API Key for the NIF data services above. More information on how to obtain API Key can be found here https://neuinfo.org/about/webservices
```